calcom/cal.com
# Stripe automatic payment method

## What does this PR do?

1. Deleting `payment_method_types` in Stripe Payments lets clients to manage payments methods by them - they know  best.
2. Introducing `automatic_payment_methods` (see [the docs](https://www.youtube.com/watch?v=5lS6WdT2GUI)) improves the payment methods - Stripe will automatically propose payment methods based on currency/locale.

Fixes #5263 

This video from the [Stripe Developers](https://www.youtube.com/watch?v=5lS6WdT2GUI&ab_channel=StripeDevelopers)  visually shows the difference


## Type of change


## Required actions ❗ 
Let connected accounts manage their payments methods:

> Stripe recommends that you allow the owners of your platform’s connected accounts to customize their own payment methods from the Dashboard. If you enable this option, then each Standard connected account can log in to their Dashboard and view their [Payment methods](https://dashboard.stripe.com/settings/payment_methods) page.

Pleas see and follow the instructions here: https://stripe.com/docs/connect/automatic-payment-methods?connect-payment-integration-type=checkout&connect-charge-type=direct&connect-account-type=standard#allow-connected-accounts-manage

## How should this be tested?
This changes the flow on the payment page from Stripe.

- Check if other payment methods are available on payments page of connected account
- Check if specific payments methods are available on payments page of connected account after customazing them under connected account payment methods

## Additional
I wasn't able to publish self hosted version to test the custom payment methods from Stripe (they block it from local) thus preview instance and the change from **Required actions** would help a lot.
